frontend/test/metabase/internal/__snapshots__/components.unit.spec.js.snap

```diff
@@ -447,6 +447,194 @@ exports[`CheckBox should render "Yellow" correctly 1`] = `
 </div>
 `;
 
+exports[`CollapseSection should render "Collapsed by default" correctly 1`] = `
```

Comment:
This is a snapshot generated from the local unit test run, it should be committed anyway